Streetcode/Streetcode.BLL/DTO/Team/CreateTeamMemberDTO.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Streetcode.BLL.DTO.Media.Images;
+
+namespace Streetcode.BLL.DTO.Team
+{
+    public class CreateTeamMemberDTO
+    {
+        public int Id { get; set; }
+        public string Name { get; set; }
```

Comment:
Do we need ID field on create DTOs? It seems like database should handle it

Reply:
You're right! It is old commit from develop, however it will be trimmed by Valeriy's changes anyway, so there is no need to delete this line here